packages/story-editor/src/components/colorPicker/basicColorList.js

```diff
@@ -44,26 +44,66 @@ const focusStyle = css`
     )};
 `;
 
-const SwatchList = styled.div.attrs({
-  role: 'listbox',
-})`
-  display: flex;
-  max-width: 100%;
-  flex-wrap: wrap;
-  gap: 6px;
+const Wrapper = styled.div`
+  display: inline-block;
+`;
+
+const SwatchList = styled.div.attrs({ role: 'listbox' })`
+  width: 100%;
+  display: inline;
+
+  > div {
+    vertical-align: top;
+
+    /* Add a grid similar to css gap in flexbox */
+    padding: 2px 3px 0 3px;
+    padding-bottom: 0;
+    /* Remove padding from swatches on left edge */
+    :nth-child(6n + 1) {
+      padding-left: 0;
+    }
+    /* Remove padding from swatches on right edge */
+    :nth-child(6n) {
```

Comment:
We can use `COLOR_PRESETS_PER_ROW` instead of using `6` directly anywhere where applicable.